src/core/NotePlayHandle.cpp

```diff
@@ -610,9 +610,9 @@ int NotePlayHandleManager::s_size;
 
 void NotePlayHandleManager::init()
 {
-	s_available = MM_ALLOC<NotePlayHandle*>( INITIAL_NPH_CACHE );
+	s_available = new NotePlayHandle*[INITIAL_NPH_CACHE];
 
-	auto n = MM_ALLOC<NotePlayHandle>(INITIAL_NPH_CACHE);
+	auto n = static_cast<NotePlayHandle *>(std::malloc(sizeof(NotePlayHandle) * INITIAL_NPH_CACHE));
```

Comment:
It is so strange that `NotePlayHandle` is not copyable, otherwise I would've said use `new NotePlayHandle[INITIAL_NPH_CACHE]`.

Reply:
Same, but because it doesn't have a default constructor, just using `new[]` won't compile.